visualization/Carte.css

```
@@ -0,0 +1,42 @@
+*{
+    margin:auto;
+
+}
+
+body{
+    background-image: url("../img/background mapm.png");
+  
+    
+}
```

TU ne veux pas plutot fixer la taille de l'image de background pour que ça prenne toute la window?
Si tel est le cas tu peux faire cette modif
```suggestion
body{
    background-image: url("../img/background mapm.png");
    background-size: cover;
    background-attachment: fixed;
}
```

---

j'ignore de quoi tu me parles je n'ai pas ce changement de mon côté

---

ah si c'est bon, mais j'ai supprimé la map ca n'avait pas de sens